components/script/dom/bindings/trace.rs

```diff
@@ -313,6 +313,18 @@ unsafe impl<T: JSTraceable> CustomTraceable for ServoArc<T> {
     }
 }
 
+unsafe impl<T: JSTraceable> CustomTraceable for std::sync::Arc<T> {
```

Comment:
As said elsewhere, let's just use 
```
#[ignore_malloc_size_of = "Arc"]
#[no_trace]
```

and remove this. Thanks for having looked into this anyway...